src/Bridges/Constraint/ltgt_to_interval.jl

```diff
@@ -0,0 +1,87 @@
+# The code here is mostly copied from the flip_sign.jl code for FlipSignBridge and GreaterToLessBridge
+
+"""
+    AbstractToIntervalBridge{T, S1, F, G}
+
+Bridge a `G`-in-`S1` constraint into an `F`-in-`Interval` constraint by multiplying
+the function by `+1` and taking the point reflection of the set across the
+origin. The flipped `F`-in-`S` constraint is stored in the `constraint`
+field by convention.
+"""
+abstract type AbstractToIntervalBridge{
+    T, S1<:MOI.AbstractSet, S2<:MOI.AbstractSet,
+    F<:MOI.AbstractFunction, G<:MOI.AbstractFunction} <: SetMapBridge{T, S2, S1, F, G} end
+
+# The function map is the identity
+map_function(::Type{<:AbstractToIntervalBridge{T}}, func) where {T} = MOIU.operate(+, T, func)
+# The map is an involution
+inverse_map_function(BT::Type{<:AbstractToIntervalBridge}, func) = map_function(BT, func)
+# The map is symmetric
+adjoint_map_function(BT::Type{<:AbstractToIntervalBridge}, func) = map_function(BT, func)
+# The map is a symmetric involution
+inverse_adjoint_map_function(BT::Type{<:AbstractToIntervalBridge}, func) = map_function(BT, func)
+
+function MOI.delete(model::MOI.ModelLike, bridge::AbstractToIntervalBridge, i::IndexInVector)
+    func = MOI.get(model, MOI.ConstraintFunction(), bridge.constraint)
+    idx = setdiff(1:MOI.output_dimension(func), i.value)
+    new_func = MOIU.eachscalar(func)[idx]
+    set = MOI.get(model, MOI.ConstraintSet(), bridge.constraint)
+    new_set = MOI.update_dimension(set, MOI.dimension(set) - 1)
+    MOI.delete(model, bridge.constraint)
+    bridge.constraint = MOI.add_constraint(model, new_func, new_set)
+end
+
+function MOI.modify(model::MOI.ModelLike, bridge::AbstractToIntervalBridge,
+                    change::MOI.ScalarCoefficientChange)
+    MOI.modify(
+        model, bridge.constraint,
+        MOI.ScalarCoefficientChange(change.variable, -change.new_coefficient))
+end
+function MOI.modify(model::MOI.ModelLike, bridge::AbstractToIntervalBridge,
+                    change::MOI.MultirowChange{T}) where T
+    new_coefficients = Tuple{Int64, T}[
+        (index, -coef) for (index, coef) in change.new_coefficients]
+    MOI.modify(model, bridge.constraint,
+               MOI.MultirowChange(change.variable,
+                                  new_coefficients))
+end
+
+"""
+    GreaterToIntervalBridge{T, F<:MOI.AbstractScalarFunction, G<:MOI.AbstractScalarFunction} <:
+        AbstractToIntervalBridge{T, MOI.GreaterThan{T}}
+
+Transforms a `G`-in-`GreaterThan{T}` constraint into an `F`-in-`Interval{T}`
+constraint.
+"""
+struct GreaterToIntervalBridge{T, F<:MOI.AbstractScalarFunction, G<:MOI.AbstractScalarFunction} <:
+    AbstractToIntervalBridge{T, MOI.GreaterThan{T}, MOI.Interval{T}, F, G}
+    constraint::CI{F, MOI.Interval{T}}
+end
+map_set(::Type{<:GreaterToIntervalBridge}, set::MOI.GreaterThan) = MOI.Interval(set.lower, Inf)
```

Comment:
Use the infinite value of type `T`, `Inf isa Float64`

Reply:
Done. I used `typemax`. I see there is a `get_bounds` function, but I just thought it was simpler to use `typemax` and `typemin` directly